### PR TITLE
Fixes #26790 - Support ES6 in Uglifier

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

**Comment by ares** [comment]
From https://github.com/lautis/uglifier/issues/127

> Thank you, that worked! - Is there any reason why I wouldn't want to enable ES6 syntax or why this feature isn't on by default?

> lautis (Owner)
> ES6 is not enabled in the master branch of UglifyJS. I've considered more or less beta, so it's not enabled by default.

Though I don't see better option atm. So :+1:  from me, if no one objects including Mr. Jenkins, I'll merge.

**Review by ares — DISMISSED** [review]

File: config/environments/production.rb [hunk 1]
```diff
@@ -22,7 +22,9 @@
   config.public_file_server.enabled = true
 
   # Compress JavaScripts and CSS.
-  config.assets.js_compressor = :uglifier
+  # config.assets.js_compressor = :uglifier
```
Comment (timogoebel) [inline config/environments/production.rb]: Do you mind removing the commented code?

**Comment by xprazak2** [comment]
I removed the comment

**Review by timogoebel — APPROVED** [review]

**Comment by tbrisker** [comment]
A bit late to the party, but shouldn't the correct fix be not to use ES6 syntax in the asset pipeline in the openscap plugin instead of changing the build process? @xprazak2

**Comment by tbrisker** [comment]
https://github.com/theforeman/foreman_openscap/pull/389 should fix the issue in openscap

**Comment by tbrisker** [comment]
reverting to fix https://projects.theforeman.org/issues/26800.

**Comment by timogoebel** [comment]
I guess I'm late to the party, but in `foreman_wreckingball` we have this code. Works quite nicely.

I hoped we would be able to have this in core so plugins can use es6 syntax while the whole webpack situation gets resolved.

https://github.com/dm-drogeriemarkt/foreman_wreckingball/blob/2b9847b50528ae475a4b9ff69e5506128da6de44/config/environments/production.rb#L5-L7

**Pull request merged**